### ISSUE TITLE
Route shape could include platforms

### ISSUE THREAD

**Issue by prhod** [CanalTP/osm-transit-extractor#9]
For the moment, there is no filter on the role of the objects included in a relation to define the shape of the route.
This behaviour is to be analyzed to avoid unexpected linestrings in the shape (like a stop platform). 
The solution would be a check on the role with `.filter(|refe| refe.role.is_empty())` : 
```rust
osm_relation.refs
    .iter()
    .filter(|refe| refe.role.is_empty())
    .filter_map(|refe| obj_map.get(&refe.member))
    .filter_map(osmpbfreader::OsmObj::way)
    .map(|way| osm_way_to_vec(obj_map, way))
    .collect()
```

**Comment by prhod** [comment]
For instance, there is the cas of the Nantes tramway : 
https://www.openstreetmap.org/relation/2116451#map=15/47.1990/-1.5395
including https://www.openstreetmap.org/way/158055032

**Comment by nlehuby** [comment]
It has been fixed via #46

**Comment by prhod** [comment]
thanks !
